prime-angular-frontend/src/app/modules/phsa-labtech/phsa-labtech.routes.ts

```diff
@@ -1,5 +1,5 @@
 export class PhsaLabtechRoutes {
-  public static PHSA_LABTECH = 'phsa-labtech';
+  public static PHSA_LABTECH = 'phsa-eforms';
```

Comment:
should we change the variable name to `PHSA_EFORMS`?

Reply:
Sure, but then I'd want to rename the entire module.  So we could do that.